skimage/registration/_masked_register_translation.py

```diff
@@ -15,31 +15,27 @@
 from .._shared.fft import fftmodule, next_fast_len
 
 
-def masked_register_translation(
-        src_image,
-        target_image,
-        src_mask,
-        target_mask=None,
-        overlap_ratio=3 / 10):
-    """
-    Masked image translation registration by masked normalized
+def masked_register_translation(reference_image, moving_image,

```

Comment:
Do we keep this function in the public API? If not it should be renamed `_masked_register_translation`.

Reply:
Since it is in the `registration._masked_register_translation` module and it is not exposed in `registration.__init__.py`, I don't consider that it is in the public API. Is it?

Reply:
No it's not (I was checking that I was not missing something). Then your stylistic convention is to prepend non-exposed functions with an underscore, so `_masked_register_translation`.

Reply:
@emmanuelle, I renamed `masked_register_translation` to `_masked_phase_cross_correlation` for consistency with the new name.

Reply:
I see in `skimage.feature._canny.py` the `smooth_with_function_and_mask` that suffers from the same drawback =p.
But this is fortunately fixed in #4342, any review is welcome ;)

Reply:
There’s actually lots of internal modules where the functions don’t have a leading underscore... I don’t think we have adhered to this policy very strictly at all.